src/qibolab/couplers.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass, field
+from typing import Dict, Optional, Union
+
+from qibolab.channels import Channel
+from qibolab.qubits import Qubit, QubitId
+
+CouplerId = Union[str, int]
+"""Type for Coupler names."""
+
+
+@dataclass
+class Coupler:
+    """Representation of a physical coupler.
+
+    Coupler objects are instantiated by :class:`qibolab.platforms.platform.Platform`
+    but they are passed to instrument designs in order to play pulses.
+
+    Args:
+        name (int, str): Qubit number or name.
+
+        flux (:class:`qibolab.platforms.utils.Channel`): Channel used to
+            send flux pulses to the qubit.
+        Other characterization parameters for the coupler, loaded from the runcard.
+    """
+
+    name: CouplerId
+
+    # TODO: I think is not needed
+    frequency: int = 0
```

Comment:
If it is not needed maybe we can remove.

Reply:
It like qubit:
```
bare_resonator_frequency: int = 0
Ec: int = 0
Ej: int = 0
g: int = 0
```
They are not actually needed but can be used I think

Reply:
If they are not needed, I would avoid. Same on both sides.